validation/src/test/scala/hmda/validation/rules/lar/macro/Q073Spec.scala

```diff
@@ -0,0 +1,61 @@
+package hmda.validation.rules.lar.`macro`
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.AggregateEditCheck
+import hmda.validation.rules.lar.`macro`.MacroEditTypes.LoanApplicationRegisterSource
+import org.scalacheck.Gen
+
+class Q073Spec extends MacroSpec {
+
+  val numOfLoanApplications = config.getInt("hmda.validation.macro.Q073.numOfLoanApplications")
+  val denialMultiplier = config.getDouble("hmda.validation.macro.Q073.numOfLarsMultiplier")
+
+  def irrelevantLar(lar: LoanApplicationRegister) = {
+    val relevantLoan = lar.loan.copy(propertyType = 2, loanType = 2, purpose = 1)
+    lar.copy(actionTakenType = 1, loan = relevantLoan)
+  }
+  def relevantLar(lar: LoanApplicationRegister) = {
+    val relevantLoan = lar.loan.copy(propertyType = 2, loanType = 2, purpose = 1)
+    lar.copy(actionTakenType = 6, loan = relevantLoan)
+  }
+
+  val irrelevantAmount: Gen[Int] = Gen.chooseNum(1, numOfLoanApplications - 1)
+  val relevantAmount: Gen[Int] = Gen.chooseNum(numOfLoanApplications, 10000)
+
+  property(s"be valid if fewer than $numOfLoanApplications specified loans") {
+    forAll(irrelevantAmount) { (x) =>
```

Comment:
Most other macro specs don't use `forAll` to run with randomly generated numbers of loans, they all just choose a few values that test the edge cases of the specific edit. 

I like that these specs cover more of the possible cases. I just want to make sure that it's okay that they don't follow the same pattern as the others, and that they'll be slower, since they're testing hundreds of cases instead of a few.

(for the record: I don't think you should have to change them to single tests; I just wanted to put that decision in the open)